src/frontends/pytorch/src/op/stft.cpp

```diff
@@ -0,0 +1,92 @@
+namespace pytorch {
+namespace op {
+
+using namespace ov::op;
+
+OutputVector translate_stft(const NodeContext& context) {
+    // schema: aten::stft(Tensor self, int n_fft, int? hop_length=None, int? win_length=None, Tensor? window=None, bool
+    // normalized=False, bool? onesided=None, bool? return_complex=None) -> Tensor
+    //
+    // Note: aten::stft doesn't have "center" and "pad_mode" attrs like torch.stft, so the number of the inputs is lower
+    // and index of any input after the "window" is smaller accordingly
+
+    num_inputs_check(context, 2, 8);
+
+    auto input = context.get_input(0);
+    auto n_fft = context.get_input(1);
+
+    ov::Output<ov::Node> hop_length;
+    if (!context.input_is_none(2)) {
+        hop_length = context.get_input(2);
+    } else {
+        // Defualt floor(n_fft / 4)
+        const auto four = context.mark_node(std::make_shared<ov::op::v0::Constant>(ov::element::i32, Shape{}, 4));
+        const auto four_cast = context.mark_node(std::make_shared<ov::op::v1::ConvertLike>(four, n_fft));
+        hop_length = context.mark_node(std::make_shared<ov::op::v1::Divide>(n_fft, four_cast));
+    }
+
+    ov::Output<ov::Node> win_length;
+    if (!context.input_is_none(3)) {
+        win_length = context.get_input(3);
+    } else {
+        win_length = n_fft;
+    }
+
+    ov::Output<ov::Node> window;
+    if (!context.input_is_none(4)) {
+        window = context.get_input(4);
+    } else {
+        const auto one = context.mark_node(std::make_shared<ov::op::v0::Constant>(ov::element::i32, Shape{}, 1));
+        const auto one_cast = context.mark_node(std::make_shared<ov::op::v1::ConvertLike>(one, input));
+        const auto zero = context.mark_node(std::make_shared<ov::op::v0::Constant>(ov::element::i32, Shape{1}, 0));
+        const auto win_length_cast =
+            context.mark_node(std::make_shared<ov::op::v0::Convert>(win_length, ov::element::i64));
+        const auto win_len_vec = context.mark_node(std::make_shared<ov::op::v0::Unsqueeze>(win_length_cast, zero));
+        window = context.mark_node(std::make_shared<ov::op::v3::Broadcast>(one_cast, win_len_vec));
+    }
+
+    bool normalized = false;
+    if (!context.input_is_none(5)) {
+        normalized = context.const_input<bool>(5);
+    }
+    PYTORCH_OP_CONVERSION_CHECK(!normalized,
+                                "aten::stft conversion is currently supported with normalized=False only.");
+
+    bool onesided = true;
+    if (!context.input_is_none(6)) {
+        onesided = context.const_input<bool>(6);
+    }
+    PYTORCH_OP_CONVERSION_CHECK(onesided, "aten::stft conversion is currently supported with onesided=True only.");
+
+    bool return_complex = false;
+    if (!context.input_is_none(7)) {
+        return_complex = context.const_input<bool>(7);
+    }
+    PYTORCH_OP_CONVERSION_CHECK(!return_complex,
+                                "aten::stft conversion is currently supported with return_complex=False only.");
```

Comment:
I see these 3 attributes are not supported, do we have plans to support them in future?

Reply:
that is probably required support complex numbers for pytorch frontend?

Reply:
Not really, complex number can be represented as a tensor with last dim as 2

Reply:
But it is, the last dimension of the STFT output is indeed 2, to represent real and imaginary part.